### PR TITLE
Update examples for Images in 2019-07-01 version according to the upd…

### PR DESCRIPTION
**Pull request by hyonholee**
…ate of 2019-03-01 version examples.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by hyonholee** [comment]
This PR is updating 2019-07-01 Compute API version according to the previous PR for 2019-03-01 version:  https://github.com/Azure/azure-rest-api-specs/pull/7396

I also found an issue in the linked PR, so I fixed it.

**Comment by hyonholee** [comment]
@tjprescott Could you review this PR?  Thanks.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: why are we removing the enum values, they provide a reference to possible values supported.

**Review by KrisBash — APPROVED** [review]
Signing off for ARM

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @smotwani   Since the enum is set to "modelAsString": true, AutoRest sets its type as string.  And after adding these "values" property, AutoRest throws an exception during SDK generation.  It seems "values" property works only when  "modelAsString" is false.

Comment (smotwani) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @hyonholee we had this ability to define enum set to "modelAsString": true and still define the possible values allowed/accepted for this enum which get translated to strings in SDK, it is important that we provide the possible string values for this property  as it cannot accept arbitrary string values and consumers of SDK need to aware of these string values, the reason we model enum as string is primarily to ensure backward clients don't when retrieving response for resources created by new clients and we are also able to ensure Get/Put symmetry. 
This limitation where AutoRest throws an exception during SDK generation when specifying values for enum set to modelAsString looks like a recent change and is possibly a regression from previous SDK generation logic.

Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @smotwani  Possible values are still there.  They are defined under "enum".   These "values" property is only for adding description, but somehow AutoRest does not like "values" when "modelAsString": true.

Comment (ChenTanyi) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @smotwani I think it is OK to delete these values, there is another "enum" field over this deletion to define the possible value.

**Review by ChenTanyi — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (ChenTanyi) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @smotwani I'll not merge until you approve for it.

Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @smotwani  Could you approve it?

**Comment by hyonholee** [comment]
@smotwani   I updated this PR by adding more description about storage account type.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Can we specify the remaining StorageAccountType, StandardSSD_LRS and UltraSSD_LRS.
Also can we say: For more information regarding disks supported for Windows Virtual Machines, refer to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types and for Linux Virtual Machines, refer to https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-types

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Can we provide separate links for Windows vs Linux VMs, like "For more information regarding disks supported for Windows Virtual Machines, refer to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types and for Linux Virtual Machines, refer to https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-types"

**Comment by hyonholee** [comment]
@smotwani I updated the description according to your comment.  Please approve it.  Thanks.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: nit: pls. remove the comma (,)....I believe it is not correct punctuation.

**Review by smotwani — APPROVED** [review]

**Comment by hyonholee** [comment]
@ChenTanyi Sameer approve it. Please merge this PR.  Thanks.

**Comment by ChenTanyi** [comment]
@hyonholee I think you could first response to his comment, thanks.

**Comment by hyonholee** [comment]
@ChenTanyi Sorry, I updated the PR according to Sameer's comment again.

**Pull request merged**
